Samples/Windowing/cs-wpf/wpf_packaged_app/TitleBarPage.xaml.cs

```diff
@@ -82,34 +87,80 @@ private void TitlebarCustomBtn_Click(object sender, RoutedEventArgs e)
                 _mainAppWindow.TitleBar.ButtonPressedBackgroundColor = Colors.Green;
                 _mainAppWindow.TitleBar.ButtonPressedForegroundColor = Colors.White;
 
-                //Infer titlebar height
-                int titleBarHeight = _mainAppWindow.TitleBar.Height;
-                _mainWindow.MyTitleBar.Height = titleBarHeight;
+                // Set the drag region for the custom title bar
+                SetDragRegionForCustomTitleBar(_mainAppWindow);
+            }
+            else
+            {
+                // Bring back the default titlebar
+                _mainWindow.MyTitleBar.Visibility = Visibility.Collapsed;
+
+                // Disable title bar height toggle buttons
+                this.StandardHeightBtn.IsEnabled = false;
+                this.TallHeightBtn.IsEnabled = false;
+
+                // reset the title bar to default state
+                _mainAppWindow.TitleBar.ResetToDefault();
+            }
+        }
+
+        private void SetDragRegionForCustomTitleBar(AppWindow appWindow)
+        {
+            //Infer titlebar height
+            int titleBarHeight = appWindow.TitleBar.Height;
+            _mainWindow.MyTitleBar.Height = titleBarHeight;
```

Comment:
The titleBarHeight variable is only used here, please simplify:
_mainWindow.MyTitleBar.Height = _mainAppWindow.TitleBar.Height;